python-checks/src/test/resources/checks/nonCallableCalledNoIssues.py

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
I'm not sure this needs a new test and test file. Why not simply have this case in `src/test/resources/checks/nonCallableCalled.py`?